Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.14
+RUN \
+  apt-get update \
+  && apt-get install -y libnetcdf-dev protobuf-compiler xz-utils cmake \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+RUN \
```

Comment:
If the image is still to large you _could_  do this all in one `RUN` and uninstall build dependencies before the `apt` clean-up. Not sure it is worth it though – how big is it?

Reply:
The image size is 920 MB.  The main win for now is mainly that we can use this as a base image for njord-wdb build, we need to keep most dependencies as we build fes dynamically.

Reply:
I wonder where all those MBs come from. If it were cmake and it's sub-dependencies it might be worth looking at, but I suspect it is netcdf, and that we need that at run-time..? Well, you can try it if you like.